partcad-ide-vscode/src/PartcadExplorer.ts

```
@@ -0,0 +1,316 @@
+    refresh(): void {
+        this._onDidChangeTreeData.fire();
+    }
+
+    public async exportItem(
+        exportType: string,
+        displayString: string,
+        fileExt: string,
+        itemType: string,
+        itemPkg: string,
+        itemName: string,
+        params: Object,
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)